### PR TITLE
Create an atom feed with absolute links for Mailchimp

### PR DESCRIPTION
**Pull request by ericholscher**
This will fix the issue where we have relative links in our emails.
Most feed readers do this automatically,
but not mailchimp.

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -26,12 +28,12 @@
 ]
 
 # Only build the videos on production, to speed up dev
-import os
 on_rtd = os.environ.get('READTHEDOCS') == 'True'
 on_netlify = os.environ.get('BUILD_VIDEOS') == 'True'
 on_travis = os.environ.get('TRAVIS') == 'True'
 if not on_rtd and not on_netlify and not on_travis:
     exclude_patterns.append('videos')
+REWRITE_FEED = False
```
Comment (plaindocs) [inline docs/conf.py:37]: Setting to `True` locally I get

```
File "/home/samuel/Nosync/Repositories/WTD/www/docs/_ext/atom_absolute.py", line 37, in rewrite_atom_feed
    namespace = root.nsmap.values()[0]
TypeError: 'dict_values' object does not support indexing
```

**Review by plaindocs — CHANGES_REQUESTED** [review]
Same error on Netlify. 

I tried wrapping with `list()` but that doesn't fix it.

File: docs/conf.py (hunk 1 above)
Comment (ericholscher) [reply docs/conf.py:37]: Hrm, probably a py3 issue, will poke at it.

**Comment by plaindocs** [comment]
Don't think this works:

```
&lt;li&gt;250,000 sessions (up from 225,000)&lt;/li&gt;
&lt;li&gt;450,000 page views (same as last year)&lt;/li&gt;
&lt;li&gt;&lt;a class=&quot;reference internal&quot; href=&quot;../../guide/&quot;&gt;&lt;span class=&quot;doc&quot;&gt;guide&lt;/span&gt;&lt;/a&gt;: 225,000 views&lt;/li&gt;
&lt;li&gt;&lt;a class=&quot;reference internal&quot; href=&quot;../../conf/&quot;&gt;&lt;span class=&quot;doc&quot;&gt;conference sites&lt;/span&gt;&lt;/a&gt;: 120,000 views&lt;/li&gt;
&lt;/ul&gt;
```

Also, mother of GOD, that escaped atom feed is !!!

**Comment by ericholscher** [comment]
> Also, mother of GOD, that escaped atom feed is !!!

Not sure I follow? The [feed](https://deploy-preview-765--dentist-canary-64055.netlify.com/blog/archive/tag/newsletter/atom_absolute.xml) I'm looking at looks great, with absolute links where previously they were relative?

**Review by plaindocs — APPROVED** [review]
Sorry, missed the "don't rewrite feeds locally", so was testing and getting relative feeds.

Also, probably we don't need that logic in? Could just rewrite to absolutely everywhere.

**Comment by ericholscher** [comment]
> Also, probably we don't need that logic in? Could just rewrite to absolutely everywhere.

This is mainly because of the lxml dependency. It can be a PITA to install, but if we don't care about that, I can remove the logic.

**Comment by plaindocs** [comment]
I don't feel very strongly about it, but also wasn't aware of the dependency issue.

**Comment by ericholscher** [comment]
Will ship this for now, and we can futz with it in the future if needed. 👍

**Pull request merged**